docs/reference/commandline/image_save.md

```diff
@@ -59,3 +60,44 @@ You can even cherry-pick particular tags of an image repository.
 ```console
 $ docker save -o ubuntu.tar ubuntu:lucid ubuntu:saucy
 ```
+
+### <a name="platform"></a> Save a specific platform (--platform)
+
+The `--platform` option allows you to specify which platform-variant of the
+image to save. By default, `docker save` saves all platform variants that
+are present in the daemon's image store. Use the `--platform` option
+to specify which platform variant of the image to save. An error is produced
+if the given platform is not present in the local image store.
+
+The platform option takes the `os[/arch[/variant]]` format, for example,
+`linux/amd64` or `limux/arm64/v8`. Architecture and variant are optional,
+and default to the daemon's native architecture if omitted.
+
+The following examples pulls an `alpine` image for the daemon's native platform,
+then provides a custom platform, which fails until the given platform-variant
+is pulled;
```

Comment:
nit:
```suggestion
The following example: 
- pulls an `alpine` image for the daemon's native platform (because `--platform` is omitted);
- tries to save the image while providing a custom `--platform` value, which fails until the given platform-variant is pulled;
```